runtime/doc/lsp.txt

```diff
@@ -2143,23 +2161,26 @@ start({cmd}, {cmd_args}, {dispatchers}, {extra_spawn_params})
 
     Parameters: ~
       • {cmd}                 (string) Command to start the LSP server.
-      • {cmd_args}            (table) List of additional string arguments to
-                              pass to {cmd}.
-      • {dispatchers}         (table|nil) Dispatchers for LSP message types.
-                              Valid dispatcher names are:
+      • {cmd_args}            (string[]) List of additional string arguments
+                              to pass to {cmd}.
+      • {dispatchers}         vim.lsp.rpc.Dispatchers ? Dispatchers for LSP message types. Valid
+                              dispatcher names are:
                               • `"notification"`
                               • `"server_request"`
                               • `"on_error"`
                               • `"on_exit"`
-      • {extra_spawn_params}  (table|nil) Additional context for the LSP
-                              server process. May contain:
+      • {extra_spawn_params}  vim.lsp.rpc.ExtraSpawnParams ? Additional context for the LSP server process.
+                              May contain:
                               • {cwd} (string) Working directory for the LSP
                                 server process
-                              • {env} (table) Additional environment variables
-                                for LSP server process
+                              • {detached} (boolean?) Detach the LSP server
+                                process from the current process. Defaults to
+                                false on Windows and true otherwise.
+                              • {env} (table?) Additional environment
+                                variables for LSP server process
 
     Return: ~
-        RpcClientPublic|nil Client RPC object, with these methods:
+        vim.lsp.rpc.PublicRpcClient ?|nil #Client RPC object, with these methods:
```

Comment:
```suggestion
        vim.lsp.rpc.PublicRpcClient|nil Client RPC object, with these methods:
```

Reply:
those are generated. (I, too, often fall for the fact that the generated docs are first in the list of files and make my comments there...)

Reply:
Worth investing if the original docs can be tweaked, though, to avoid that.